### DISCUSSION TITLE
Coffee Donations :)

### DISCUSSION THREAD

**Discussion by Nexow91** [jhassan8/crunchyroll-tizen#24] [General]
Hi @jhassan8 I didn't know now where to open this post or how best to write to you about github. 

I would like to support the project a little financially in the hope that the project will last for a long time and will get better and better and you will also be rewarded for your efforts and work, because you also put in some time and does good work.

I hope that is in your sense if yes you can me yes gladly if you have your Paypal email address in Github deposit so that others can send you a coffee donation. 

Many greetings

**Comment by jhassan8** [comment]
Hi, that's awesome, thank you! Actually, I'm doing it to entertain myself... I'm going to see what I can create that can be used in Argentina :D
